.github/workflows/deploy_branch_preview.yml

```diff
@@ -60,8 +65,21 @@ jobs:
             ${{ runner.os }}-
       - name: Install Packages
         run: npm install
+      - name: Get Latest Release
```

Comment:
Can we rename this step to be more explicit on its purpose, i.e. latest binary version